appeals/web/src/server/appeals/appeal-details/appellant-case/address/address.controller.js

```diff
@@ -74,6 +75,13 @@ export const postChangeSiteAddress = async (request, response) => {
 		return response.redirect(redirectUrl);
 	} catch (error) {
 		logger.error(error);
+
+		// Check if it's a validation error (400)
+		if (error instanceof HTTPError && error.response.statusCode === 400) {
+			// @ts-ignore
```

Comment:
Ideally this `ts-ignore` wouldn't be needed, just checking it's not feasible to remove?

Reply:
Struggling with this one. I know we're trying to clean out `ts-ignore`s, but thought we'd let this one slide.